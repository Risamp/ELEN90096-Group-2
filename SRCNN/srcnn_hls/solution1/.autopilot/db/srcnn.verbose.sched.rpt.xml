<profile>

<section name = "Vitis HLS Report for 'srcnn'" level="0">
<item name = "Date">Tue Oct 31 13:44:43 2023
</item>
<item name = "Version">2023.1 (Build 3854077 on May  4 2023)</item>
<item name = "Project">srcnn_hls</item>
<item name = "Solution">solution1 (Vivado IP Flow Target)</item>
<item name = "Product family">zynquplus</item>
<item name = "Target device">xck26-sfvc784-2LV-c</item>
</section>

<section name = "Performance Estimates" level="0">
<item name = "Timing">
<section name = "" level="1">
<item name = "Summary"><table name="" hasTotal="0">
<keys size="4">Clock, Target, Estimated, Uncertainty</keys>
<column name="ap_clk">10.00 ns, 7.542 ns, 2.70 ns</column>
</table>
</item>
</section>
</item>
<item name = "Latency">
<section name = "" level="1">
<item name = "Summary"><table name="" hasTotal="0">
<keys size="8">, min, max, min, max, min, max, Type</keys>
<column name="">3379702801, 3379702801, 33.797 sec, 33.797 sec, 3379702802, 3379702802, no</column>
</table>
</item>
<item name = "Detail">
<section name = "" level="1">
<item name = "Instance"><table name="" hasTotal="0">
<keys size="9">Instance, Module, min, max, min, max, min, max, Type</keys>
</table>
</item>
<item name = "Loop"><table name="" hasTotal="0">
<keys size="8">Loop Name, min, max, Latency, achieved, target, Count, Pipelined</keys>
</table>
</item>
</section>
</item>
</section>
</item>
</section>
</profile>
