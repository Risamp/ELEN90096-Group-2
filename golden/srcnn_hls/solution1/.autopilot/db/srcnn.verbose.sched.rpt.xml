<profile>

<section name = "Vitis HLS Report for 'srcnn'" level="0">
<item name = "Date">Mon Oct  9 14:51:29 2023
</item>
<item name = "Version">2023.1 (Build 3854077 on May  4 2023)</item>
<item name = "Project">srcnn_hls</item>
<item name = "Solution">solution1 (Vivado IP Flow Target)</item>
<item name = "Product family">zynquplus</item>
<item name = "Target device">xck26-sfvc784-2LV-c</item>
</section>

<section name = "Performance Estimates" level="0">
<item name = "Timing">
<section name = "" level="1">
<item name = "Summary"><table name="" hasTotal="0">
<keys size="4">Clock, Target, Estimated, Uncertainty</keys>
<column name="ap_clk">10.00 ns, 7.016 ns, 2.70 ns</column>
</table>
</item>
</section>
</item>
<item name = "Latency">
<section name = "" level="1">
<item name = "Summary"><table name="" hasTotal="0">
<keys size="8">, min, max, min, max, min, max, Type</keys>
<column name="">4829781636, 4829781636, 48.298 sec, 48.298 sec, 4829781637, 4829781637, no</column>
</table>
</item>
<item name = "Detail">
<section name = "" level="1">
<item name = "Instance"><table name="" hasTotal="0">
<keys size="9">Instance, Module, min, max, min, max, min, max, Type</keys>
</table>
</item>
<item name = "Loop"><table name="" hasTotal="0">
<keys size="8">Loop Name, min, max, Latency, achieved, target, Count, Pipelined</keys>
<column name="- VITIS_LOOP_18_1">3142040832, 3142040832, 49094388, -, -, 64, no</column>
<column name=" + VITIS_LOOP_22_2">49094385, 49094385, 192527, -, -, 255, no</column>
<column name="  ++ VITIS_LOOP_23_3">192525, 192525, 755, -, -, 255, no</column>
<column name="   +++ VITIS_LOOP_26_4">747, 747, 83, -, -, 9, no</column>
<column name="    ++++ VITIS_LOOP_27_5">81, 81, 9, -, -, 9, no</column>
<column name="- VITIS_LOOP_17_1">1215203616, 1215203616, 37975113, -, -, 32, no</column>
<column name=" + VITIS_LOOP_21_2">37975110, 37975110, 148922, -, -, 255, no</column>
<column name="  ++ VITIS_LOOP_22_3">148920, 148920, 584, -, -, 255, no</column>
<column name="   +++ VITIS_LOOP_33_6">576, 576, 9, -, -, 64, no</column>
<column name="- VITIS_LOOP_21_2">472537185, 472537185, 1853087, -, -, 255, no</column>
<column name=" + VITIS_LOOP_22_3">1853085, 1853085, 7267, -, -, 255, no</column>
<column name="  ++ VITIS_LOOP_25_4">7260, 7260, 1452, -, -, 5, no</column>
<column name="   +++ VITIS_LOOP_26_5">1450, 1450, 290, -, -, 5, no</column>
<column name="    ++++ VITIS_LOOP_33_6">288, 288, 9, -, -, 32, no</column>
</table>
</item>
</section>
</item>
</section>
</item>
</section>
</profile>
